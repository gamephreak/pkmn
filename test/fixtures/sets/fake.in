Fakey (Fake) @ Fake  
Ability: Fake  
- Fake Move 1  
- Fake Move 2  
- Fake Move 3  
- Fake Move 4  

