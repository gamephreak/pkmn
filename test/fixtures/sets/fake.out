Fakey (fake) @ fake  
Ability: fake  
- fakemove1  
- fakemove2  
- fakemove3  
- fakemove4  

